docs/ovms_quickstart.md

```diff
@@ -0,0 +1,101 @@
+# OpenVINO&trade; Model Server Quickstart
+
+To quickly start using OpenVino&trade; Model Server please execute the following steps.
+
+### Deploy Model Server
+
+The Docker image with OpenVino&trade; Model Server is available on the DockerHub. Issue the
+following command to start using it:

```

Comment:
Change to `A Docker image with OpenVINO Model Server is available on DockerHub. Use the following command to download the latest release:`

Reply:
done